src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -43,7 +48,7 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         if (!StringUtil.isNonZeroUnsignedInteger(trimmedIndex)) {
             throw new ParseException(MESSAGE_INVALID_INDEX);
         }
-        return Index.fromOneBased(Integer.parseInt(trimmedIndex));
+        return Index.fromOneBased(parseInt(trimmedIndex));
```

Comment:
Should use Integer.parseInt(trimmedIndex)), because if someone creates a parseInt(trimmedIndex) method, then this line might break the code.

Reply:
Thanks for the catch 😃